workspaces/ui-v2/src/optic-components/diffs/render/AskForCommitMessage.tsx

```diff
@@ -29,6 +27,9 @@ const useStagedChangesCount = () => {
       (diffHash) => diffHashToEndpoint[diffHash]
     ),
     ...Object.keys(context.choices.existingEndpointNameContributions),
+    ...Object.values(context.choices.existingEndpointPathContributions).map(
+      (pathContribution) => pathContribution.endpointId
```

Comment:
There's a bit of juggling / passthrough for data here - I think here is where a global store would really help to simplify these data derivations (such that you have a single source of truth - especially since our endpoints / paths / shapes are modeled as a graph)

The FE here cares about which endpoints were modified (which is not super trivial to do with the current structure)